README.md

```diff
@@ -30,3 +31,28 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+Recurrence relation for mystery():
+
+$T(n) =$ 1 if $n <= 1$ <br>
+$3T(n / 3) + 3n^2$ otherwise <br>
```

Comment:
Where does the $3n^2$ come from? How many nested loops do you have, and how often does each one run?

Reply:
This is still not correct.

Reply:
Do the loops run in constant time? The constant is just whatever n*n is....?